src/components/taskDetails/index.tsx

```diff
@@ -259,22 +229,38 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                         ]
                                     }
                                 >
-                                    {taskTitle.length ? (
-                                        taskTitle.map((title, index) => (
-                                            <Link
-                                                href={`/tasks/${id[index]}`}
-                                                key={index}
-                                            >
-                                                <li
-                                                    onClick={() =>
-                                                        navigateToTask(
-                                                            id[index]
-                                                        )
-                                                    }
-                                                >
-                                                    {title}
-                                                </li>
-                                            </Link>
+                                    {loading || fetching ? (
```

Comment:
this is not at all readable make this in to a proper if else

Reply:
done